spec/javascripts/helpers/sort_helpers_spec.jsx

```diff
@@ -0,0 +1,39 @@
+import SortHelpers from '../../../app/assets/javascripts/helpers/sort_helpers.jsx';
+
+describe('SortHelpers', function() {
+  describe('#sortByCustomEnum', function() {
+    const customEnum = ['One', 'Two', 'Three', 'Four', 'Five'];
+    const preSortData = [
+      {testElement: 'Five'},
+      {testElement: 'Four'},
+      {testElement: 'Five'},
+      {testElement: 'One'},
+      {testElement: null}
+    ];
+    
+    it('correctly sorts', function() {
+      let sortData = preSortData.slice();
+      
+      let targetData = [
+        {testElement: null},
+        {testElement: 'One'},
+        {testElement: 'Four'},
+        {testElement: 'Five'},
+        {testElement: 'Five'}
+      ];
+      
+      sortData.sort((a, b) => SortHelpers.sortByCustomEnum(a, b, 'testElement', customEnum));
+      
+      let match = true;
+
+      for(let i=0; i<preSortData.length; i++){
+        if(sortData[i]['testElement'] != targetData[i]['testElement']) {
+          match = false;
+        }
+      }
+
+      expect(match).toBe(true);
+      
+    });
+  });
+});
```

Comment:
I was having a bone-headed problem with the enum sorting so I decided to add a spec. Wasn't the code, was my calling it but decided to leave the spec in to exercise that code.